deps.cfg

```diff
@@ -0,0 +1,2 @@
+flutter_version=2.5.0-6.0.pre
```

Comment:
Please also add copyright notices for this file (as if it was a bash script.)

Reply:
Done